definitions/organization/cluster_role.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: blockchain:admin
+rules:
```

Comment:
Do we need to set permissions for this cluster role?

Reply:
No default permission for now . Permissions will be set when Organization created.